state_management/python/sdk/order-processor/app.py

```diff
@@ -0,0 +1,21 @@
+import random
+from time import sleep
+import logging
+from dapr.clients import DaprClient
+
+logging.basicConfig(level=logging.INFO)
+
+DAPR_STORE_NAME = "statestore"
+while True:
+    sleep(random.randrange(50, 5000) / 1000)
+    order = {'orderId': random.randint(1, 1000)}
+    with DaprClient() as client:
+        # Save state into the state store
+        client.save_state(DAPR_STORE_NAME, "orderId", str(order))
+        # Get state from the state store
+        result = client.get_state(DAPR_STORE_NAME, "orderId")
+        logging.info('Result after get: ' + str(result.data))
+        # Delete state from the state store
+        client.delete_state(store_name=DAPR_STORE_NAME, key="order_1")
```

Comment:
Why does this example pass key="", but the other examples do not? Makes me think it's option especially since the dapr docs showing "key=" for each state call:  https://docs.dapr.io/developing-applications/sdks/python/python-client/#save--get-application-state.


If it is optional, I would omit it.